hooks/pre-command

```diff
@@ -121,16 +33,18 @@ read_list_property() {
   done < <(env | sort)
 }
 
+get_default_image_name() {
+  echo "build-cache/${BUILDKITE_ORGANIZATION_SLUG}/${BUILDKITE_PIPELINE_SLUG}"
+}
+
 compute_tag() {
   local docker_file="$1"
   local sums
 
-  echoerr() { echo "$@" 1>&2; }
-
   echoerr '--- Computing tag'
 
   echoerr 'DOCKERFILE'
-  echoerr "+ ${docker_file}:${target:-<target>}"
+  echoerr "+ ${docker_file}:${target:-"<target>"}"
```

Comment:
Are these extra quotes required?

Reply:
No, that's muscle memory on my part. I tend to always enclose strings in quotes and variables inside {} because it's easier than remembering where it will be necessary and where it won't be necessary, compared to just reflexively doing it everywhere.